styles/widgets/android5/list.android5.less

```diff
@@ -54,15 +53,11 @@
         .background-size-prop(100%);
     }
 
-    &.dx-state-active {
+    .dx-inkripple-wave {
```

Comment:
`.dx-inkripple-wave` and `&.dx-state-active` use the same backgound color. 
You can change it to
```
&.dx-state-active,
.dx-inkripple-wave {
  background-color: @ANDROID5_LIST_ITEM_HOVER_BACKGROUND;
}
```